Gateway/Request/Recipient/KlarnaDataBuilder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Buckaroo\Magento2\Gateway\Request\Recipient;
+
+use Buckaroo\Magento2\Gateway\Request\Recipient\AbstractRecipientDataBuilder;
+
+class KlarnaDataBuilder extends AbstractRecipientDataBuilder
+{
+    protected function buildData(): array
+    {
+        return [
+            'category' => $this->getCategory(),
+            'gender' => $this->getGender(),
```

Comment:
If we add the extra values to the parent getGender method, we should add a new one here to contain only male/female values.